packages/gestalt/src/Controller.js

```diff
@@ -150,6 +151,10 @@ class Controller extends Component<Props, State> {
           shouldFocus={shouldFocus}
           triggerRect={triggerBoundingRect}
           width={width}
+          __dangerouslyIgnoreScrollBoundaryContainerSize={
+            // eslint-disable-next-line no-underscore-dangle
+            this.props.__dangerouslyIgnoreScrollBoundaryContainerSize
```

Comment:
`          __dangerouslyIgnoreScrollBoundaryContainerSize={__dangerouslySetMaxWidth === '30vh"}`

Reply:
This was made on Popover component level and propagate to down, because the component structure documented [here](https://paper.dropbox.com/doc/Popovers-History--B0B6_pXAfccwGF~0PRE7Xi1MAg-e5Yrz9jiksN0ce8c893EJ).